src/components/ContactsList/ContactsEmptyList.jsx

```diff
@@ -89,9 +101,17 @@ class ContactsEmptyList extends React.Component {
                 />
               </span>
               <span className="contacts-empty-action">
-                {!isToutaticeInstance && (
-                  <ImportGoogleButton onComplete={this.afterConnection} />
-                )}
+                <ButtonLink
+                  className="contacts-empty-button"
```

Comment:
seems to not be used

Reply:
Yes it is, but we can use cozy-ui class utilities instead